uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-balance-tab-filters.component.js

```diff
@@ -7,21 +7,23 @@ define([
 ], function (_, moment, components) {
   components.component('leaveBalanceTabFilters', {
     controller: LeaveBalanceTabFiltersController,
-    controllerAs: 'leaveBalanceTabFilters',
+    controllerAs: 'filtersVm',
```

Comment:
I want to avoid confusion with the filters property, is it ok to have filters.filters on the view?